### PR TITLE
First implementation for private timeline

### PR DESCRIPTION
**Pull request by michael1929**
WARNING: Needs a new Database file

### CODE REVIEW

File: src/main/java/at/ac/fhstp/Sniffable/datamodel/Comment.java [hunk 1]
```diff
@@ -8,6 +8,8 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
```
Comment (Thomas131) [inline src/main/java/at/ac/fhstp/Sniffable/datamodel/Comment.java:10]: Is there a @Column or something missing?

Comment (Thomas131) [reply src/main/java/at/ac/fhstp/Sniffable/datamodel/Comment.java:10]: Ups, I didn't want to mention a user, sorry @Column

File: src/main/java/at/ac/fhstp/Sniffable/datamodel/Pupdate.java [hunk 2]
```diff
@@ -10,6 +10,8 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
 
+    private long timestamp;
```
Comment (Thomas131) [inline src/main/java/at/ac/fhstp/Sniffable/datamodel/Pupdate.java:13]: same

**Review by Thomas131 — CHANGES_REQUESTED** [review]
See comments (not sure if needed). Looks good to me otherwise. Thanks for that!

**Review by Thomas131 — APPROVED** [review]

**Review by harames2001 — APPROVED** [review]
well done

**Pull request merged**